gateway/modules/schema/inspection.go

```diff
@@ -64,11 +64,9 @@ func generateInspection(dbType, col string, fields []model.InspectorFieldType, f
 		}
 
 		// default key
-		if field.FieldDefault != "" {
+		if field.FieldDefault != "" && field.AutoIncrement == "false" {
 			fieldDetails.IsDefault = true
 			if model.DBType(dbType) == model.SQLServer {
-				// replace (( or )) with nothing e.g -> ((9.8)) -> 9.8
-				field.FieldDefault = strings.Replace(strings.Replace(field.FieldDefault, "(", "", -1), ")", "", -1)
```

Comment:
Are you sure about this?

Reply:
Yes, To confirm you can check the SQL Server query in describe table, The filtering logic is implemented in the query itself